src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -62,6 +89,22 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         #region Authorization and Credentials
 
         /// <summary>
+        /// gets or sets the AllowUnencryptedAuthentication property
+        /// </summary>
+        [Parameter]
+        public virtual SwitchParameter AllowUnencryptedAuthentication { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Authentication property used to determin the Authentication method for the web session. 
+        /// Authentication does not work with UseDefaultCredentials.
+        /// Authentication over unencrypted sessions requires AllowUnencryptedAuthentication.
+        /// Basic: Requires Credential
+        /// OAuth/Bearer: Requires Token
+        /// </summary>
+        [Parameter]
+        public virtual WebAuthenticationType Authentication { get; set; }
```

Comment:
Should we assign default `WebAuthenticationType.None`?

Reply:
I don't think ther is a need, The enum is never null so its creation here automatically sets it to `None` since it is the first enum value... at least.. if i understand everything I read correctly. If it wasn't automatically None, all of the many tests would be failing. Unless setting it here provides some kind of emit somewhere. I didin't see any other enum parameters setting a default value.

Reply:
If internal logic is based on `None` we should initialize explicitly to `None` (Can we sort the enum or add new element on first position?). Another way is to being based on Authentication.IsPresent.

Reply:
fixed.